web/templates/stream_settings/stream_creation_form.hbs

```diff
@@ -40,8 +40,15 @@
             </div>
         </div>
         <div class="settings-sticky-footer">
-            <button class="button small white rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-            <button class="finalize_create_stream button small sea-green rounded" type="submit">{{t "Create" }}</button>
+            <div class="settings-sticky-footer-left">
+                <button id="create_stream_back_tab" class="button small sea-green rounded hide">{{t "Configure settings" }}</button>
+            </div>
+
+            <div class="settings-sticky-footer-right">
+                <button class="create_stream_cancel button small white rounded" data-dismiss="modal">{{t "Cancel" }}</button>
+                <button class="finalize_create_stream button small sea-green rounded hide" type="submit">{{t "Create" }}</button>
+                <button id="create_stream_next_tab" class="button small sea-green rounded">{{t "Continue to add subscribers" }}</button>
```

Comment:
I think we should use better IDs here instead of using `next_tab` and `back_tab`  and have more specific names like `stream_creation_go_to_subscribers`, etc.